.github/workflows/codeql-analysis.yml

```diff
@@ -12,13 +12,20 @@
 name: "CodeQL"
 
 on:
-#  push:
-#    branches: [ master ]
-#  pull_request:
-#    # The branches below must be a subset of the branches above
-#    branches: [ master ]
-  schedule:
-    - cron: '29 18 * * 0'
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - '**/*.md'
+      - '**/*.rst'
+      - '**/*.txt'
```

Comment:
nit: the comment in the commit message is likely sufficient for a developer to understand why we also need to run this on any push to master, but also couldn't hurt to add it in the workflow file itself.

Reply:
good point, updated